rdbms/src/test/java/org/spine3/server/storage/jdbc/aggregate/query/Given.java

```diff
@@ -38,7 +38,7 @@
 class Given {
 
     private static Logger loggerMock = null;
-    private static final IdColumn<String> idColumnMock = mock(IdColumn.StringIdColumn.class);
+    private static final IdColumn<String> ID_COLUMN_QUERY_SETTER_MOCK = mock(IdColumn.StringIdColumn.class);
```

Comment:
Please if this is a [real constant](this: http://links.teamdev.com/constant-names).